src/library/scala/concurrent/ExecutionContext.scala

```diff
@@ -121,23 +121,27 @@ trait ExecutionContextExecutorService extends ExecutionContextExecutor with Exec
  */
 object ExecutionContext {
   /**
-   * The explicit global `ExecutionContext`. Invoke `global` when you want to provide the global
-   * `ExecutionContext` explicitly.
+   * The global [[ExecutionContext]] This default `ExecutionContext` implementation is backed by a work-stealing thread
```

Comment:
Needs a full-stop.  Or lower case "This" and connect it.

Reply:
`wherewith`? Groping for a connector.